src/Uno.UWP/ApplicationModel/DataTransfer/DataPackage.cs

```diff
@@ -93,6 +93,146 @@ public DataPackageView GetView()
 				_data,
 				ResourceMap.ToImmutableDictionary(),
 				(id, op) => OperationCompleted?.Invoke(this, new OperationCompletedEventArgs(id, op)));
+
+		/// <summary>
+		/// Determines if the given URI/URL is considered a WebLink.
+		/// This determination is done based on whether the scheme starts with 'http' or 'https'.
+		/// </summary>
+		/// <remarks>
+		/// This method is intended for use during integration with other platforms.
+		/// Therefore, it uses a string instead of a Uri or native URL class.
+		/// </remarks>
+		/// <param name="uri">The URI to determine if it is a WebLink.</param>
+		/// <returns>True if the given URI/URL is considered a WebLink; otherwise, false.</returns>
+		internal static bool IsUriWebLink(string? uri)
+		{
+			uri = uri?.Trim();
+
+			if (string.IsNullOrEmpty(uri) == false)
+			{
+				if (uri!.StartsWith("http", StringComparison.InvariantCultureIgnoreCase) ||
+					uri!.StartsWith("https", StringComparison.InvariantCultureIgnoreCase))
+				{
+					return true;
+				}
+			}
+
+			return false;
+		}
+
+		/// <summary>
+		/// Separated a single URI/URL into separate URI's for WebLink and ApplicationLink (Uri format is deprecated).
+		/// This is useful for converting a platform-specific URI/URL into formats supported by the <see cref="DataPackage"/>.
+		/// </summary>
+		/// <param name="uri">The platform-specific URI/URL.</param>
+		/// <param name="webLink">
+		/// The <see cref="StandardDataFormats.WebLink"/> format Uri intended for use in a <see cref="DataPackage"/>.
+		/// Null will be returned if the URI/URL is not in the <see cref="StandardDataFormats.WebLink"/> format.
+		/// </param>
+		/// <param name="applicationLink">
+		/// The <see cref="StandardDataFormats.ApplicationLink"/> format Uri intended for use in a <see cref="DataPackage"/>.
+		/// Null will be returned if the URI/URL is not in the <see cref="StandardDataFormats.ApplicationLink"/> format.
+		/// </param>
+		internal static void SeparateUri(string? uri, out string? webLink, out string? applicationLink)
```

Comment:
Codacy found an issue: [Either change this parameter type to 'System.Uri' or provide an overload which takes a 'System.Uri' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6364607)